voluncheer/map/urls.py

```diff
@@ -2,6 +2,7 @@
 
 from map import views
 
+app_name = "map"
```

Comment:
What does this give us? Also if we are going to have a top-level constant it should be appropriately named using UPPER_SNAKE_CASE.